src/js/actions/app-state-actions.js

```diff
@@ -116,9 +116,15 @@ export function setServerURL(url) {
 }
 
 export function showErrorNotification(message) {
-    return {
-        type: SHOW_ERROR_NOTIFICATION,
-        message
+    return (dispatch) => {
+        setTimeout(() => {
+            dispatch(hideErrorNotification(message));
```

Comment:
Here, it means any error notifications will be dismissed, even if it didn't originate in the same call to `showErrorNotification`.

We could send an id when dispatching the `SHOW_ERROR_NOTIFICATION` action which would have to be sent with its associated `HIDE_ERROR_NOTIFICATION` action to make sure the right notification is dismissed. Thoughts @lemieux?